### PR TITLE
Refactor trust-proxy tests

### PR DESCRIPTION
**Pull request by trivikr**
#### Checklist

- [x] run `npm run test`
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: test/trust-proxy.test.js [hunk 1]
```diff
@@ -5,23 +5,46 @@ const test = t.test
 const sget = require('simple-get').concat
 const fastify = require('..')
 
+const sgetForwardedRequest = (app, forHeader, path) => {
+  sget(
+    {
```
Comment (mcollina) [inline test/trust-proxy.test.js]: can you please move { to the previous line and reformat the rest of the indentation?

Comment (trivikr) [reply test/trust-proxy.test.js]: Done

**Comment by trivikr** [comment]
@jsumners  This refactoring will make it easy to write tests for other use cases (when number/string is passed as a value for trustProxy param) https://github.com/fastify/fastify/blob/acf39508fc6653112dc8198269222789339d4e51/fastify.js#L227-L235

**Comment by trivikr** [comment]
Any update on this request?

**Comment by jsumners** [comment]
I'm not blocking this. Was just wanting clarification on the intent.

**Pull request merged**